library.gradle

```diff
@@ -26,11 +26,13 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
     }
+
     sourceSets {
-        main.java.srcDirs += 'src/main/kotlin'
-        main.java.srcDirs += 'src/main/java'
-        test.java.srcDirs += 'src/test/java'
+        unityIAP {
+            java.srcDirs = ['src/latestDependencies/java']
+        }
```

Comment:
Hmm should we add a comment that this needs to be removed once we finish bc5 support and remove `bc5supportDev`?